tests/utils/v2Data.js

```diff
@@ -65,6 +67,75 @@ function generateFakeEvents(start, stop, count) {
     return range(count, eventsEvery).map(i => makeEvent(Math.floor(start + i)));
 }
 
+
+class ExpressResponseStub {
+    constructor() {
+        this._status = null;
+        this._body = null;
+        this._redirect = null;
+    }
+
+    status(code) {
+        this._status = code;
+        return this;
+    }
+
+    sendStatus(code) {
+        this._status = code;
+    }
+
+    send(body) {
+        this._body = body;
+        return this;
+    }
+
+    redirect(url) {
+        this._redirect = url;
+        return this;
+    }
+}
+
+const stubLogger = {
```

Comment:
Maybe use sinon to create the stubs?

Reply:
I'd like to leave this rather dependency agnostic so tests not otherwise using sinon don't have to worry about side effects. Additionally stubbing it with sinon wouldn't net us much in the way of brevity or simplicity.